rest/src/commonMain/kotlin/service/RestClient.kt

```diff
@@ -35,6 +35,10 @@ public class RestClient(requestHandler: RequestHandler) : RestService(requestHan
     // topics
     public val application: ApplicationService = ApplicationService(requestHandler)
 
+    // monetization
+    public val entitlement: EntitlementService = EntitlementService(requestHandler)
+    public val skus: SkuService = SkuService(requestHandler)
```

Comment:
```suggestion
    public val sku: SkuService = SkuService(requestHandler)
    public val entitlement: EntitlementService = EntitlementService(requestHandler)
```
they are ordered the other way around, also singular `sku` instead of `skus`